plugins/mcreator-localization/help/default/gamerule/category.md

```diff
@@ -0,0 +1,3 @@
+This is the GameRule's category. It's used for the classification of the GameRules depending on what they do.
+
+Note: Only used in Minecraft 1.16.4+
```

Comment:
Change to `NOTE: Only available in Minecraft 1.16.x and higher` to have same format as other notes for this

Reply:
Why did you even set that to `Minecraft 1.16.x`? I mean, people can't create mods in MCreator in 1.16 or 1.16.1 or 1.16.2 or 1.16.3

Reply:
It is to be more general, one can make a 1.16.3 generator if they feel like it.

Reply:
well in `item/immune_to_fire.md`, it is set to `NOTE: Only available for Minecraft 1.16.4+`

Reply:
I did not write this and it obviously slipped through review. Why are we arguing this, exactly?

If you find it in other notes, feel free to open new PR where you can fix them to keep notes streamlined.

Reply:
> Why are we arguing this, exactly?

We're not, I wanted to show you it isn't written `NOTE: Only available in Minecraft 1.16.x and higher` everywhere else.

Reply:
![image](https://user-images.githubusercontent.com/16374228/103478840-b5469880-4dc9-11eb-8d65-1c9a56f81937.png)

The current state in the master branch seems mostly translations need to streamline this at the current point